.achecker.yml

```diff
@@ -5,5 +5,3 @@ failLevels:
 
 policies:
   - WCAG_2_1
-
-ruleArchive: preview
```

Comment:
Just for clarity, would it be better to have `ruleArchive: lastest` in the file even when it is already the default?

Reply:
I like that — I'll be sure to carry that practice into `ibm-cloud-cognitive`, thank you!